A

michaellindsay23 opened the pull request.

michaellindsay23 committed:
a

TheGamer1002 committed:
Merge branch 'master' into a

deepsource-autofix[bot] committed:
style: format code with Ktlint

This commit fixes the style issues introduced in 972faab according to the output
from Ktlint.

Details: https://github.com/Bearbots6964/FRC2024-offseason/pull/2

TheGamer1002 commented on src/main/java/frc/robot/subsystems/vision/VisionSubsystem.kt:
```diff
@@ -0,0 +1,25 @@
+package frc.robot.subsystems
```
package frc.robot.subsystems.vision

TheGamer1002 reviewed (changes requested):
needs to build lmao

TheGamer1002 committed:
feat(thing): fix

deepsource-autofix[bot] committed:
style: format code with Ktlint

This commit fixes the style issues introduced in 6c4c746 according to the output
from Ktlint.

Details: https://github.com/Bearbots6964/FRC2024-offseason/pull/2
